examples/server_hardware.go

```diff
@@ -22,44 +23,95 @@ func main() {
 		apiversion,
 		"*")
 
+	fmt.Println("-----------------------------")
+	fmt.Println("Add Single Rack server to the appliance:")
+	rackServer := ov.ServerHardware{
+		Hostname:           "<serverIp>",
+		Username:           "<username>",
+		Password:           "<password>",
+		Force:              false,
```

Comment:
have you validated the force option..i mean by setting it true...it should not break anything